touchforms/formplayer/tests/test_experiment.py

```diff
@@ -0,0 +1,54 @@
+from django.test import SimpleTestCase
+from touchforms.formplayer.experiments import formplayer_string_compare, formplayer_compare
+
+
+class ExperimentTest(SimpleTestCase):
+
+    control = {}
+    candidate = {}
+
+    def testExperimentStringCompare(self):
+        # session_id will never be equal, should be ignored
+        assert formplayer_string_compare("123", "234", current_key="session_id")
+        # same with output XML (for the time being)
+        assert formplayer_string_compare("123", "234", current_key="output")
+
+        assert formplayer_string_compare(0, "false", current_key="repeatable")
+        assert formplayer_string_compare(1, "true", current_key="repeatable")
+        assert not formplayer_string_compare(1, "false", current_key="repeatable")
+        assert not formplayer_string_compare(0, "true", current_key="repeatable")
+
+    def testExperimentDictCompare(self):
```

Comment:
this might be more concises if there was a `setUp` function that called reset and then a separate test for each case you're testing